packages/next/src/client/components/react-dev-overlay/internal/container/StaticIndicator.tsx

```diff
@@ -7,17 +7,7 @@ export function StaticIndicator({ dispatcher }: { dispatcher?: Dispatcher }) {
   return (
     <Toast className="nextjs-static-indicator-toast-wrapper">
       <LightningBolt />
-      <span>
-        Statically rendered page.{' '}
-        <a
-          onClick={(e) => e.stopPropagation()}
-          href="https://nextjs.org/docs/app/api-reference/next-config-js/devIndicators"
-          target="_blank"
-          rel="noreferrer"
-        >
-          Learn more
-        </a>
-      </span>
+      <span>Static route </span>
```

Comment:
Is the extra space here intentional? looks like we have margin on the close icon so not sure that it's needed

Reply:
Just left for mental spacing the margin is main though